src/components/DocsLink/DocsLink.js

```diff
@@ -0,0 +1,40 @@
+import { useConfig } from '@dhis2/app-runtime'
+import i18n from '@dhis2/d2-i18n'
+import { IconButton } from 'material-ui'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { getDocsKeyForSection } from '../../pages/sections.conf'
+
+const getDocsVersion = ({ major, minor, tag }) => {
+    if (tag === 'SNAPSHOT') {
+        return 'master'
+    }
+    return `${major}${minor}`
+}
+
+const DocsLink = ({ sectionKey }) => {
+    const { serverVersion } = useConfig()
+    const docsVersion = getDocsVersion(serverVersion)
+    const docsKey = getDocsKeyForSection(sectionKey)
+
+    return (
+        <IconButton
+            className="helper-icon"
+            iconStyle={{ color: '#276696' }}
```

Comment:
Maybe we should use one of the colors from our ui theme here, for consistency. If that works with this material ui component that is.